assignments/closure.js

```diff
@@ -1,10 +1,40 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+function animalNoise(sound) {
+	let soundMade = sound;
+  	function animal(soundMade) {
+      if(soundMade == 'quack') {
+        return 'duck'
+      } else {
+        return 'who knows what kind of animal that is!'
+      }
+    }
+    return animal;
+  };
+  
+let animalType = animalNoise();
+
+console.log(animalType('quack'));
+console.log(animalType('meow'));
```

Comment:
I really like this example! Note though, that because `soundMade` is the parameter for the `animal` function, it actually shadows the `soundMade` variable in the animalNoise closure, so animal never makes use of the closure! DM me to discuss more if you would like, as it's a useful thing to understand.